helpers/connectionHelper.js

```diff
@@ -186,6 +186,23 @@ function getFeed(){
   return getResults(SQL, []);
 }
 
+function getObjectIdByItemName(itemName){
+  const SQL = 'SELECT objectId FROM object WHERE objectName = ?';
```

Comment:
not too sure about SQL syntax myself, but I believe this would be "SELECT * FROM object WHERE objectName = ?" since as I mentioned in another comment, we really just want every part of the object.

Reply:
Also, can we have it be where the object name contains parts of the string that is entered, rather than equaling it?